src/components/hooks/use-livechat.js

```diff
@@ -52,10 +53,11 @@ export const useLivechat = () => {
             clearInterval(cookie_interval)
             clearTimeout(script_timeout)
         }
-    }, [])
+    }, [runScripts])
```

Comment:
isn't `isBrowser` a dependency in this hook?

Reply:
no it is not, they only wrote a isBrowser condition inside the useEffect.